app/models/doctor.rb

```diff
@@ -0,0 +1,8 @@
+class Doctor < ActiveRecord::Base
+  validates :name, presence: true
+  validates_length_of :name, maximum: 35
```

Comment:
Prefer the new style validations `validates :column, length: value` over `validates_length_of`.